MultimodalQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -379,19 +385,29 @@ curl -X POST \
 
 8. MegaService
 
+Test the MegaService with a text query:
 ```bash
 curl http://${host_ip}:8888/v1/multimodalqna \
     -H "Content-Type: application/json" \
     -X POST \
     -d '{"messages": "What is the revenue of Nike in 2023?"}'
 ```
 
+Test the MegaService with an audio query:
 ```bash
 curl http://${host_ip}:8888/v1/multimodalqna  \
     -H "Content-Type: application/json"  \
     -d '{"messages": [{"role": "user", "content": [{"type": "audio", "audio": "UklGRigAAABXQVZFZm10IBIAAAABAAEARKwAAIhYAQACABAAAABkYXRhAgAAAAEA"}]}]}'
 ```
 
+Test the MegaService with a text and image query:
+```bash
+curl http://${host_ip}:8888/v1/multimodalqna \
+    -H "Content-Type: application/json" \
+    -d  '{"messages": [{"role": "user", "content": [{"type": "text", "text": "Green bananas in a tree"}, {"type": "image_url", "image_url": {"url": "http://images.cocodataset.org/test-stuff2017/000000004248.jpg"}}]}]}'
+```
```

Comment:
This works, but I get the response "There is no video segments retrieved given the query!" Is it supposed to find one of the uploaded items and return something more meaningful?